libethcore/Ethash.h

```diff
@@ -66,7 +66,8 @@ class Ethash
 
 		h256 boundary;
 		h256 headerHash;	///< When h256() means "pause until notified a new work package is available".
-		h256 seedHash;
+		h256 seedHash; /// LTODO: IS this needed now that we use the block number instead?
```

Comment:
for the work package, either this or the number - from within cpp-ethereum they are interchangeable.